src/renderer/cursor_renderer/mod.rs

```diff
@@ -286,6 +299,10 @@ impl CursorRenderer {
         )
             .into();
 
+        // The width for the outline of a block cursor should be the same as the width of
+        // a vertical cursor.
+        let outline_width = cursor_dimensions.x * DEFAULT_CELL_PERCENTAGE * 2.0;
```

Comment:
Would it matter to make this a configurable option? This is more of a selfish request, I like these kinds of outlines to be very thin.

To introduce the new config option, just add it to the `CursorSettings` struct, you can access it here through `settings` which is already in scope then.